### PR TITLE
[dashboard]:  add a bottom modal sheet in dashboard

### PR DESCRIPTION
**Pull request by Jibea**
add to dashboard screen a modal screen with an exemple of widgets

### CODE REVIEW

File: SurvivorNext/Pages/DashboardPage.js [hunk 1]
```diff
@@ -1,12 +1,107 @@
...
 
 export default function DashboardPage() {
+
+  const [isModalVisible, setModalVisible] = useState(false);
+  // ref
+  const bottomSheetRef = useRef(null);
+
+  const handleIndicatorPress = useCallback(() => {
+    console.log('handleIndicatorPress');
+    // Fermez la modal lorsque vous cliquez en dehors d'elle
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/DashboardPage.js]:
```suggestion
    // Close the modal when you click outside
```

**Review by MathisCaugant — APPROVED** [review]
Good for me !

**Review by TurpaultM — APPROVED** [review]
it's great

File: SurvivorNext/Pages/DashboardPage.js [hunk 2]
```diff
@@ -1,12 +1,107 @@
...
+  },
 
-const styles = StyleSheet.create({})
\ No newline at end of file
+  itemContainer: {
+    width: '50%',
+    padding: 12,
+  },
+  overlay: {
+    backgroundColor: '#E5E7E6', // Overlay semi-transparent
```
[the remark this answers is not in the document]
Comment (TurpaultM) [reply SurvivorNext/Pages/DashboardPage.js]: this commentary is great, it's not an obligation to delete it

**Pull request merged**